docs/analyzer/user_guide.md

```diff
@@ -1018,12 +1023,6 @@ analyzer arguments:
                         Run analysis only with the analyzers specified.
                         Currently supported analyzers are: clangsa, clang-
                         tidy.
-  --add-compiler-defaults
```

Comment:
Please, find other occurrences of this flag in this file and remove those ones too.